test/functional/dispatch-handle-run.c

```diff
@@ -164,7 +163,7 @@ static void register_test_function(void)
 
   info.con->closed = true;
   info.con->msgid = 1;
-  info.con->cc.pluginkey = pluginkey;
+  memcpy(info.con->cc.pluginkeystring, pluginkey, PLUGINKEY_STRING_SIZE);
```

Comment:
The pluginkeystring is an array and not of type `char *`. So copying seems legit and necessary.